sound/hybrid_font.h

```diff
@@ -232,18 +232,22 @@ class HybridFont : public SaberBase, public Looper {
     if (loop) hum_player_->PlayLoop(loop);
    }
 
-  void PlayMonophonic(Effect* f, Effect* loop, float xfade = 0.003f)  {
-    PlayMonophonic(f->RandomFile(), loop, xfade);
+  void PlayMonophonic(Effect* f, Effect* loop, float xfade = 0.003f) {
+      Effect::FileID id = f->RandomFile();
```

Comment:
I just checked in a change that does the getfollowing thing in RandomFile, which applies to all the play functions in one go.

Reply:
oh nice. Ok let me adjust all this then, because I didn't know you were doing that.
Might be a tomorrow thing.

Reply:
> I just checked in a change that does the getfollowing thing in RandomFile, which applies to all the play functions in one go.

ProffieOS/modes/settings_menues.h:219:48: error: 'OFF_IDLE' was not declared in this scope;

Reply:
Nope, that was easy. Just reverted. Done.